### PR TITLE
Cleanup and refactoring

### PR DESCRIPTION
**Pull request by devanych**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ✔️
| Fixed issues  | #1

### CODE REVIEW

**Comment by devanych** [comment]
Waiting: https://github.com/yiisoft/db/pull/215

File: config/common.php [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+use Psr\Log\LoggerInterface;
+use Yiisoft\Db\Connection\ConnectionInterface;
+use Yiisoft\Log\Logger;
+use Yiisoft\Log\Target\Db\DbTarget;
+
+return [
+    LoggerInterface::class => static fn (DbTarget $dbTarget) => new Logger([$dbTarget]),
+
+    DbTarget::class => static fn (ConnectionInterface $db) => new DbTarget($db),
```
Comment (samdark) [inline config/common.php:13]: Wouldn't it be done automatically if definition is missing?

File: config/params.php [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<?php
+
+declare(strict_types=1);
+
+return [
+    'yiisoft/yii-db-migration' => [
+        'createNamespace' => 'Yiisoft\Log\Target\Db\Migration',
```
Comment (samdark) [inline config/params.php:7]: How would that affect regular application migrations? Would these still work?

**Comment by samdark** [comment]
👍

**Pull request merged**